src/modules/muorb/slpi/uORBProtobufChannel.cpp

```diff
@@ -78,11 +172,20 @@ static void *test_runner(void *)
 
 int px4muorb_orb_initialize(fc_func_ptrs *func_ptrs, int32_t clock_offset_us)
 {
-	// These function pointers will only be non-null on the first call
-	// so they must be saved off here
-	if (func_ptrs != nullptr) { muorb_func_ptrs = *func_ptrs; }
+	hrt_set_absolute_time_offset(clock_offset_us);
 
-	PX4_INFO("px4muorb_orb_initialize called");
+	if (! px4muorb_orb_initialized) {
+		uORB::Manager::initialize();
+		uORB::Manager::get_instance()->set_uorb_communicator(
+			uORB::ProtobufChannel::GetInstance());
+
+
+		if (func_ptrs != nullptr) { muorb_func_ptrs = *func_ptrs; }
```

Comment:
If those function pointers are null then everything fails! Also check that each of the individual pointers are not null.